#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("magnify_distance", double_t, 0, "Magnify distance", 0.2, 0.0, 10.0)
exit (gen.generate (PACKAGE, "jsk_pcl_ros_utils", "PolygonMagnifier"))
